docs/whats_new/v0.6.2.md

```diff
@@ -0,0 +1,40 @@
+# Version 0.6.2
+
+v0.6.2 is a security and bug fix release.
+
+This release is backward compatible to 0.6.0 and 0.6.1. As always, only load previously trained models using the same version of AutoGluon that they were originally trained on.
```

Comment:
"This release is backward compatible to 0.6.0 and 0.6.1.":

I never tested that, nor do I suspect did anyone else in any module test that. I suspect it doesn't hold True in all cases. I don't think we want to claim backwards compatibility.

Reply:
Or do you mean API is same? In that case yes. But you shouldn't be loading a v0.6.1 predictor in v0.6.2, we didn't test that.

Reply:
Will remove that statement. Got similar statement from priror 0.4.3